### PR TITLE
[Prow CI chain Enhancement] update hypershift-aws-create to support QE team e2e test

### PR DESCRIPTION
**Pull request by LiangquanLi930**
add `instance-type` configuration to support QE continue run e2e test for openshift release.

### CODE REVIEW

File: ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml [hunk 1]
```diff
@@ -34,6 +34,9 @@ chain:
     - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
       default: "SingleReplica"
       documentation: "Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+    - name: HYPERSHIFT_INSTANCE_TYPE
```
Comment (bryan-cox) [inline ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml:37]: nit: May want to add `_x86` or `_AMD64` or something like that to the name. Sometime we may have multi-arch hosted clusters/NodePools. There is a PR to implement that is on hold. The instance type will matter if we are using ARM over x86.

File: ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml [hunk 2]
```diff
@@ -34,6 +34,9 @@ chain:
...
+      default: "m5.large"
```
Comment (enxebre) [inline ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml:38]: any reason why this is changing default from m5.xlarge -> m5.large?

Comment (LiangquanLi930) [reply ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml:38]: Sorry, I will change it back^^

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-main-periodics-4.13-conformance-aws-ovn

**Pull request merged**